functions/sg/deploy.sh

```diff
@@ -0,0 +1,39 @@
+# Deploy the submit_sg_workload cloud function
+# usage: bash deploy.sh TRIGGER_BUCKET_NAME
+# ex: bash deploy.sh broad-gotc-dev-wfl-sg-test-inputs
+
+TRIGGER_BUCKET_NAME=${1}
+TRIGGER_EVENT=${2:-"google.storage.object.finalize"}
+REGION=${3:-"us-central1"}
+
+# gotc-prod
+if [ "${TRIGGER_BUCKET_NAME}" == "???" ]; then
+    GCLOUD_PROJECT="broad-gotc-prod-storage"
+    SA_EMAIL="sg-submission-fn-non-prod@broad-gotc-prod-storage.iam.gserviceaccount.com"
+    _WFL_URL="https://gotc-prod-wfl.gotc-prod.broadinstitute.org"
+    _CROMWELL_URL="https://cromwell-gotc-auth.gotc-prod.broadinstitute.org/"
+    _WORKLOAD_PROJECT="sg-prod"
```

Comment:
Are these the actual project names or placeholder values? I'm wondering because I don't see them listed in gcloud 🤔

Reply:
project gets passed to WFL's `project` field. It felt weird still calling the environment variable `WFL_ENVIRONMENT` [like the aou fn does](https://github.com/broadinstitute/wfl/blob/main/functions/aou/main.py#L57) because we don't ever actually have sg environments anywhere (we use the wgs ones)

Reply:
Do you think I should rename this?

Reply:
Oh! I didn't see `GCLOUD_PROJECT ` a few lines up 🤦‍♀️ I think the name makes sense